projects/strater/index.js

```diff
@@ -92,8 +94,18 @@ async function tvl(api) {
   const stapearlUSDTAmount =
     (stapearlLpAmount * stapearlReserveY) / stapearlLpSupply;
 
+  // saving vault
+  const savingVaultcObj = await sui.getObject(ST_SBUCK_VAULT_OBJECT_ID)
+  let savingVaultTVL = Number(savingVaultcObj.fields.free_balance) + Number(savingVaultcObj.fields.time_locked_profit.fields.locked_balance) + Number(savingVaultcObj.fields.time_locked_profit.fields.unlocked_balance)
+  const strategies = savingVaultcObj.fields.strategies.fields.contents
+  for(const strategy of strategies){
+    const botrrowedAmount = Number(strategy.fields.value.fields.borrowed)
+    savingVaultTVL += botrrowedAmount
```

Comment:
hi @Jarekkkkk thanks for the update, it is mostly good except for this part, we cant count borrowed as tvl, it is leverage. 
![image](https://github.com/user-attachments/assets/e8ea1af3-acbd-4c6f-8592-58f434ed9a38)

In this example, we can count only the deposited 10 SUI as tvl not the borrowed 10 SUI

Reply:
> hi @Jarekkkkk thanks for the update, it is mostly good except for this part, we cant count borrowed as tvl, it is leverage. ![image](https://private-user-images.githubusercontent.com/99907941/348946934-e8ea1af3-acbd-4c6f-8592-58f434ed9a38.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExMDgwMzAsIm5iZiI6MTcyMTEwNzczMCwicGF0aCI6Ii85OTkwNzk0MS8zNDg5NDY5MzQtZThlYTFhZjMtYWNiZC00YzZmLTg1OTItNThmNDM0ZWQ5YTM4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE2VDA1Mjg0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcwODYyZDQ1YWM1NDFkNTNmNDQwMWE2Njc1MDI3ODdkNDU5YTI2ZTAzMWRmZmM3MmU2ODE1NWNlMWQ5OGMxNjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OBlod_c73RTbzVWqOQH6gfuHFKhaXPkdqLfyKtXe9NU)
> 
> In this example, we can count only the deposited 10 SUI as tvl not the borrowed 10 SUI

Hi @g1nt0ki, thx for the reply

actually the vault you're referring to is different from the one we add.

[saving_vault_page](https://www.strater.xyz/vault/auto-compound?type=ST-SBUCK)

above is the vault page on our site, kindly take a look. the TVL only accounts for the deposited BUCK, there aren't any assets being borrwed under the hood. 

let me know if there's any questions !

thank !

Reply:
ah, all good then, thanks for the update.